test/distributed/fsdp/test_shard_utils.py

```diff
@@ -185,3 +186,20 @@ def get_offsets(tensor, shard):
         uneven_sharded_tensor.gather(0, output)
         if self.rank == 0:
             self.assertEqual(tensor, output)
+
+    @skip_if_lt_x_gpu(2)
+    def test_create_chunk_sharded_tensor(self):
+        for size in ((1,), (1, 6), (12,), (12, 6), (25,), (25, 6)):
```

Comment:
Do you think testing at most 2 dimensions suffices? I guess it should be fine since you are only sharding along dim 0 so as long as there is a second dim, it should be okay.

Reply:
Unless we change the implementation to chunk in a different dimension, 2 dimensions test should be enough.